test/e2e/case19_rep_policy_placement_test.go

```diff
@@ -1,241 +0,0 @@
-		Expect(afterTotal - beforeTotal).Should(Equal(2))
-	})
-	It("should reconcile twice when managed1 and managed2 are created by placementRule", func() {
-		// Now, connected cluster is managed1
-		beforeString := utils.GetMetrics("controller_runtime_reconcile_total",
-			`controller=\"replicated-policy\"`,
-			`,result=\"success\"`,
-		)[0]
-		beforeTotal, err := strconv.Atoi(beforeString)
-		Expect(err).ShouldNot(HaveOccurred())
-		By("Patching gvrPlacementBinding placementRef")
-		plb := utils.GetWithTimeout(
-			clientHubDynamic,
-			gvrPlacementBinding,
-			case19PolicyName+"-pb",
-			testNamespace,
-			true,
-			defaultTimeoutSeconds,
-		)
-		plb.Object["placementRef"].(map[string]interface{})["name"] = case19PolicyName + "-plr"
-		plb.Object["placementRef"].(map[string]interface{})["kind"] = "PlacementRule"
-		plb.Object["placementRef"].(map[string]interface{})["apiGroup"] = gvrPlacementRule.Group
-		_, err = clientHubDynamic.Resource(gvrPlacementBinding).Namespace(testNamespace).Update(
-			context.TODO(),
-			plb, metav1.UpdateOptions{},
-		)
-		Expect(err).ToNot(HaveOccurred())
-		afterString := utils.GetMetrics("controller_runtime_reconcile_total",
-			`controller=\"replicated-policy\"`,
-			`,result=\"success\"`,
-		)[0]
-		By("Managed1 deleted, Should be only 1 reconcile")
-		afterTotal, err := strconv.Atoi(afterString)
-		Expect(err).ShouldNot(HaveOccurred())
-		Expect(afterTotal - beforeTotal).Should(Equal(1))
-
-		By("Get controller_runtime_reconcile_total number before change placementrule")
-		beforeString = utils.GetMetrics("controller_runtime_reconcile_total",
-			`controller=\"replicated-policy\"`,
-			`,result=\"success\"`,
-		)[0]
-		beforeTotal, err = strconv.Atoi(beforeString)
-		Expect(err).ShouldNot(HaveOccurred())
-
-		By("Patching test-policy-plr with placementRule of cluster managed1, managed2")
-		plr := utils.GetWithTimeout(
-			clientHubDynamic,
-			gvrPlacementRule,
-			case19PolicyName+"-plr",
-			testNamespace,
-			true,
-			defaultTimeoutSeconds,
-		)
-		plr.Object["status"] = utils.GeneratePlrStatus("managed1", "managed2")
-		_, err = clientHubDynamic.Resource(gvrPlacementRule).Namespace(testNamespace).UpdateStatus(
-			context.TODO(),
-			plr, metav1.UpdateOptions{},
-		)
-		Expect(err).ShouldNot(HaveOccurred())
-
-		By("Check replicate policy changed")
-		plc := utils.GetWithTimeout(
-			clientHubDynamic, gvrPolicy, testNamespace+"."+case19PolicyName, "managed1", true, defaultTimeoutSeconds,
-		)
-		Expect(plc).ToNot(BeNil())
-		plc2 := utils.GetWithTimeout(
-			clientHubDynamic, gvrPolicy, testNamespace+"."+case19PolicyName, "managed2", true, defaultTimeoutSeconds,
-		)
-		Expect(plc2).ToNot(BeNil())
-		afterString = utils.GetMetrics("controller_runtime_reconcile_total",
-			`controller=\"replicated-policy\"`,
-			`,result=\"success\"`,
-		)[0]
-		afterTotal, err = strconv.Atoi(afterString)
-		Expect(err).ShouldNot(HaveOccurred())
-		Expect(afterTotal - beforeTotal).Should(Equal(2))
```

Comment:
This is the one I was thinking could alternatively just be:
```
Expect(afterTotal - beforeTotal).Should(Equal(Or(2,3)))
```

Reply:
It should be 2. because managed1, managed2 added. This happens only "minimum"